api/src/domain/user/updateOperatingPhase.ts

```diff
@@ -69,7 +70,8 @@ const getNewPhase = ({
 
   if (
     businessPersona === "FOREIGN" &&
-    (foreignBusinessType === "REMOTE_WORKER" || foreignBusinessType === "REMOTE_SELLER")
+    (determineForeignBusinessType(foreignBusinessTypeIds) === "REMOTE_WORKER" ||
+      determineForeignBusinessType(foreignBusinessTypeIds) === "REMOTE_SELLER")
```

Comment:
[sand] how do you feel about setting 2 variables `isRemoteWorker` and `isRemoteSeller` just above this and then using them in the if?